tools/autodeployment/deploy.sh

```diff
@@ -24,42 +22,35 @@ cd telescope
 # Set NGINX FILE + ENV_FILE Params
 if [ $1 = 'production' ]
 then
-  ENV_FILE=env.production
+  ENV_FILE=config/env.production
 elif [ $1 = 'staging' ]
 then
-  sed -i 's/telescope\./dev\.telescope\./g' nginx.conf
-  ENV_FILE=env.staging
+  sed -i 's/telescope\./dev\.telescope\./g' config/nginx.conf
```

Comment:
We'll probably need to do this as well for `api.telescope...` vs `dev.api.telescope...`